client/my-sites/email/email-management/email-home.tsx

```diff
@@ -30,15 +30,21 @@ import type { ReactNode } from 'react';
 
 import './style.scss';
 
-const ContentWithHeader = ( props: { children: ReactNode } ) => {
+interface ContentWithHeaderProps {
+	children: ReactNode;
+	className?: string;
+}
+const ContentWithHeader = ( props: ContentWithHeaderProps ) => {
 	const translate = useTranslate();
+	const { children, className } = props;
```

Comment:
```suggestion
const ContentWithHeader = ( { children, className }: ContentWithHeaderProps ) => {
	const translate = useTranslate();
```
Nit, but if we destructure `props` anyway, why not do it like this?

Reply:
Done!